website/content/en/status/report-2023-04-2023-06/kde.adoc

```diff
@@ -0,0 +1,60 @@
+=== KDE on FreeBSD
+
+Links: +
+link:https://freebsd.kde.org/[KDE FreeBSD] URL: link:https://freebsd.kde.org/[] +
+link:https://community.kde.org/FreeBSD[KDE Community FreeBSD] URL:link:https://community.kde.org/FreeBSD[]
+
+Contact: Adriaan de Groot <kde@FreeBSD.org>
+
+The KDE on FreeBSD project packages CMake, Qt, and software from the KDE Community, for the FreeBSD ports tree.
+The software includes a full desktop environment called KDE Plasma (for both X11 and Wayland) and hundreds of applications that can be used on any FreeBSD machine.
+
+The KDE team (kde@) is part of desktop@ and x11@, building the software stack to make FreeBSD beautiful and usable as a daily-driver graphical desktop workstation.
+The notes below describe *mostly* ports for KDE, but also include items that are important for the entire desktop stack.
+
+==== Infrastructure
+
+* Qt5 ports had various updates:
+  * package:devel/qt5-webengine[] was repaired when building with Clang 16. This is preparation for the upcoming release of FreeBSD 14.
```

Comment:
I think your intention here is to have an unordered sublist: please use `**` instead, `   *` does not seem to work.
The same holds below for the other occurrences.

Maybe "this is in preparation"?

Please put one sentence per line, even in unordered lists.

Reply:
- Yes, intended as an unordered list.
- Sure, "This is in preparation" works for me.
- Can you split it up? I am unsure of how the indentation would work then, and I don't have time to fix it right now.